src/extension.ts

```diff
@@ -231,3 +240,63 @@ async function notifyEditorServiceDisabled(context: vscode.ExtensionContext) {
     context.globalState.update(suppressEditorServicesDisabled, true);
   }
 }
+
+async function notifyIfNewPDKVersion(context: vscode.ExtensionContext, settings:IAggregateConfiguration) {
+  const suppressPDKUpdateCheck = 'suppressPDKUpdateCheck';
+  const dontCheckAgainNotice = "Don't check again";
+  const viewPDKDownloadPage = "More info";
+
+  // if (context.globalState.get(suppressPDKUpdateCheck, false)) {
```

Comment:
nit: can probably delete this commented code

Reply:
Actually, that was supposed to be there. That honors the user telling us to stop checking for PDK versions. Although...there is no way to ask to be notified again...

Reply:
>  Although...there is no way to ask to be notified again

That's not a usual workflow though. "Please pester me!"

Reply:
Uncommented and verified it works on my machine :)

Reply:
This is still commented out.  Did you forget to push your changes?